js/components/portal-dashboard/question-area.tsx

```diff
@@ -77,4 +80,12 @@ export class QuestionArea extends React.PureComponent<IProps>{
       </div>
     );
   }
+
+  private renderImageQuestionPrompt = (drawingPrompt: string) => {
```

Comment:
argument should be `prompt`, not `drawingPrompt`

Reply:
And you could pass a boolean here like 'showDivider` so you only show the divider if both prompts exist.